amlb/runners/aws.py

```diff
@@ -208,7 +208,7 @@ def run(self, tasks: Union[str, List[str]] = None, folds: Union[int, List[int]]
                 task_names = None if tasks is None else [task_def.name for task_def in task_defs]
                 job = self._make_aws_job(task_names, folds)
                 results = self._run_jobs([job])
-                return self._process_results(results, task_name=tasks)
+                return self._results_summary(self._process_results(results))
```

Comment:
Since `self._process_results(results)` can do some work, I'd prefer it on a separate line.
```suggestion
                scoreboard = self._process_results(results)
                return self._results_summary(scoreboard)
```